src/spec_assign.cpp

```diff
@@ -396,6 +400,11 @@ struct spell_trainer spelltrainers[] = {
                             {10013, SPELL_LIGHTNINGBOLT, "Lightning Bolt", 0, 50},
                             {10013, SPELL_BALLLIGHTNING, "Ball Lightning", 0, 50},
                             {10013, SPELL_CLOUT, "Clout", 0, 50},
+                            {10013, SPELL_LASER, "Laser", 0, 50},
+                            {10013, SPELL_NOVA, "Nova", 0, 50},
+                            {10013, SPELL_STEAM, "Steam", 0, 50},
+                            {10013, SPELL_THUNDERBOLT, "Thunderclap", 0, 50},
+                            {10013, SPELL_THUNDERCLAP, "Thunderclap", 0, 50},
```

Comment:
Did you also want to add 'smokecloud' and 'splash'?

Reply:
Yes.. and I forgot to in the new push. RIP